docs/tutorial.md

```diff
@@ -12,7 +12,7 @@ part. :)
 
 The data in your program has types: Lua is a high-level language, so each
 piece of data stored in the memory of the Lua virtual machine has a type:
-number, string, function, boolean, userdata, thread, nil or table.
+integer, number, string, function, boolean, userdata, thread, nil or table.
```

Comment:
This section in referring to Lua, and integer is not a separate Lua type. It is a subtype for _some_ Lua versions (but it is a type for Teal regardless of Lua version). Probably best to skip integers here.